runtime/doc/news.txt

```diff
@@ -116,6 +116,9 @@ The following changes may require adaptations in user config or plugins.
   upstream tree-sitter and Helix to make it easier to share queries. The full
   list is documented in |treesitter-highlight-groups|.
 
+• The "match" table returned by |Query:iter_matches()| and that is used in
+  |treesitter-predicates| now correctly maps capture IDs to a list of matching
```

Comment:
```suggestion
  |treesitter-predicates| and |treesitter-directives| now correctly maps capture IDs to a list of matching
```

Reply:
As a side note, I wonder if we could make the error message more helpful (rather than unexpected nil error) for better downstream migration experience, but it would be at the cost of small performance loss.